gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/service/modules/orchestration/AzkabanClient.java

```diff
@@ -130,10 +137,12 @@ private void initializeClient() throws AzkabanClientException {
 
   private void initializeSessionManager() throws AzkabanClientException {
     if (sessionManager == null) {
-      this.sessionManager = new AzkabanSessionManager(this.httpClient,
-                                                      this.url,
-                                                      this.username,
-                                                      this.password);
+      try {
+        this.sessionManager = new AzkabanSessionManager(this.httpClient, this.url, this.username, this.password);
+      }
+      catch(Exception e) {
+        log.error("Failed to initialize session manager due to: " + e);
```

Comment:
Same thing here, use the log.error throwable argument like I mentioned above.
https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#error-java.lang.CharSequence-java.lang.Throwable-